include/solvers/simulation_parameters.h

```diff
@@ -142,10 +145,10 @@ class SimulationParameters
     multiphysics.parse_parameters(prm);
 
     // Check consistency of parameters parsed in different subsections
-    if (multiphysics.free_surface && physical_properties.number_of_fluids != 2)
+    if (multiphysics.VOF && physical_properties.number_of_fluids != 2)
       {
         throw std::logic_error(
-          "Inconsistency in .prm!\n with free surface = true\n use: number of fluids = 2");
+          "Inconsistency in .prm!\n with VOF = true\n use: number of fluids = 2");
       }
   }
```

Comment:
This was fixed in my last PR, but if you want to keep the duplicate tests why not...

Reply:
I did not understand the comment, I kept it as it was. should I remove or change it?